packages/eslint-plugin-nx/src/rules/enforce-module-boundaries.ts

```diff
@@ -33,7 +33,7 @@ import {
   isComboDepConstraint,
 } from '../utils/runtime-lint-utils';
 import { AST_NODE_TYPES, TSESTree } from '@typescript-eslint/utils';
-import { TargetProjectLocator } from 'nx/src/utils/target-project-locator';
+import { TargetProjectLocator } from 'nx/src/plugins/js/project-graph/build-dependencies/target-project-locator';
```

Comment:
We are using TargetProjectLocator also for linter, where we do not `build dependencies` so perhaps placing this in the `nx/src/plugins/js/project-graph` makes more sense?

Reply:
I think the eslint plugin is making a deep import so it doesn't really justify changing the export path for that. I don't really like that it depends on this class either.. But that's out of scope for this.